### ISSUE TITLE
Mongolfier issue: MySQL connect error: ECONNREFUSED

### ISSUE THREAD

**Issue by icewing02** [npm/npm#10289]
Hey There,

I have tried researching how to resolve this issue but I cannot find any that answers the issue. For you to understand more about the issue, check this out:

```
bash$ node ./node_modules/.bin/mongolfier
load templates...
load script: /some_path/table_name.json
open mysql connection...
mysql connection is opened!
open mongo connection...
mongo connection is opened without authentication!
process before hooks...
no before hooks...
process collections...
process 1 collections...
prepare collection: table_name
process collection:table_name
mysql query on error: { [Error: connect ECONNREFUSED]
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  fatal: true }
failed to mirate! { [Error: connect ECONNREFUSED]
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  fatal: true }
```

Is there anything I am missing? My configuration seems to be correct. Thanks a lot.
Looking forward for your response.

Best Regards,
Jay

**Comment by KenanY** [comment]
@icewing02 Hmm this does not look like an issue with `npm` to me. I think it would be best if you opened this issue over at the issue tracker for the module you are having issues with.

**Comment by ehsalazar** [comment]
@icewing02 I replied to your Twitter DM as well, but will cross post here. 

Typically your best resource for package specific questions, is the folks that published said package. In this case, I took a look at mongolfier’s codebase on github and it appears as though the project has been abandoned. The [mongolfier README](https://github.com/iolo/mongolfier) gives a deprecation warning. 

I'll close this issue for now, but will be happy to reopen if needed.
